modules/lvms-troubleshooting-investigating-a-pvc-stuck-in-the-pending-state.adoc

```diff
@@ -4,17 +4,20 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="investigating-a-pvc-stuck-in-the-pending-state_{context}"]
-= Investigating a PVC stuck in the Pending state
+= Investigating a PVC stuck in the pending state
```

Comment:
```suggestion
= Investigating a PVC stuck in the `Pending` state
```

Pending is a specific state that is observable, which is why its upper cased

Reply:
Updated as per the suggestion.